### PR TITLE
Fix SendFakeSyncVar to maingame 11.x 

### PR DESCRIPTION
**Pull request by sanyae2439**
Updated for Mirror change.
(and add comments for SendFakeSyncObject)

### CODE REVIEW

**Comment by PintTheDragon** [comment]
ty sanyae ❤️

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 1]
```diff
@@ -292,40 +304,44 @@ private static int GetComponentIndex(NetworkIdentity identity, Type type)
...
-                behaviour = behaviorOwner.NetworkBehaviours[i];
-                if (behaviour.GetType() == targetType)
+                if (behaviorOwner.NetworkBehaviours[i].GetType() == targetType)
                 {
-                    dirty |= 1UL << i;
-                    if (behaviour.syncMode == SyncMode.Observers)
-                        dirty_o |= 1UL << i;
+                    behaviour = behaviorOwner.NetworkBehaviours[i];
+                    behaviorDirty = (byte)i;
                 }
```
Comment (PintTheDragon) [inline Exiled.API/Extensions/MirrorExtensions.cs:317]: Can a break be added here? I assume there won't be multiple items in behaviorOwner.NetworkBehaviours with the same type, right?

Comment (sanyae2439) [reply Exiled.API/Extensions/MirrorExtensions.cs:318]: Yes, thats more efficient, changed it.

**Review by PintTheDragon — APPROVED** [review]
LGTM

**Comment by sanyae2439** [comment]
Found Base classes issue, so i fix it.

**Pull request merged**